lib/commons/color/get-background-color.js

```diff
@@ -124,20 +131,37 @@ function _getBackgroundColor(elm, bgElms, shadowOutlineEmMax) {
 }
 
 /**
- * Determine if element is partially overlapped, triggering a Can't Tell result
+ * Checks whether a node fully encompasses a set of rects.
  * @private
- * @param {Element} elm
- * @param {Element} bgElm
- * @param {Object} bgColor
+ * @param {Element} node
+ * @param {NodeRect[]} rects
  * @return {Boolean}
  */
-function elmPartiallyObscured(elm, bgElm, bgColor) {
-  var obscured =
-    elm !== bgElm && !visuallyContains(elm, bgElm) && bgColor.alpha !== 0;
-  if (obscured) {
-    incompleteData.set('bgColor', 'elmPartiallyObscured');
+function fullyEncompasses(node, rects) {
+  rects = Array.isArray(rects) ? rects : [rects];
+
+  const nodeRect = node.getBoundingClientRect();
+  const style = window.getComputedStyle(node);
+  const overflow = style.getPropertyValue('overflow');
```

Comment:
Should this handle overflow-x and overflow-y separately?

Reply:
For what reason?

Reply:
Sorry, never mind. I didn't know this, but it seems you did. So if I set overflow-x to `auto`, then overflow-y is also `auto`, even if it was set to something else. Weird.